docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/paragraph--cgov-card-external.html.twig

```diff
@@ -17,4 +23,4 @@
 {% set image = content.field_image_promotional %}
 
 {## Render Card ##}
-{{cardMacros.featureCard(title, description, image, url)}}
+{{ attribute(cardMacros, cardType, [title, description, image, url])}}
```

Comment:
Yeah, this DEFINITELY needs a comment. That is not a normal way to use attribute in any Drupal examples and caused some head scratching. (Neat usage though...)